src/Codec/JVM/ASM/Code/Instr.hs

```diff
@@ -32,7 +32,8 @@ data InstrState =
              , isLabelTable    :: LabelTable
              , isLastBranch    :: LastBranch
              , isRunAgain      :: Bool
-             , isNextLabel     :: Int }
+             , isNextLabel     :: Int 
+             , isLineNumber    :: Maybe Int }
```

Comment:
Shouldn't `isLineNumber` be a Map since you can have many entries for a given method body?